src/renderer/bootstrap.tsx

```diff
@@ -34,6 +35,7 @@ export async function bootstrap(App: AppComponent) {
   rootElem.classList.toggle("is-mac", isMac);
 
   extensionLoader.init();
+  extensionDiscovery.init();
```

Comment:
Maybe I am just weird, but why are we asking if this might be done in `main` since this in under `src/renderer`. Why not just call `initRenderer()` directly?

Reply:
Well, calling initRenderer could work, but that's an implementation detail I guess. Now the caller doesn't need to know that the class has different init method for different "environments", making it easier to call the initialization.

Reply:
Fair I guess. I do know that I have always been slightly annoyed that we use the "same" classes for both `main` and `renderer` but I guess this is not the PR to do that work in.